src/dist/replication/lib/replica_stub.cpp

```diff
@@ -889,6 +889,41 @@ void replica_stub::on_query_replica_info(const query_replica_info_request &req,
     resp.err = ERR_OK;
 }
 
+void replica_stub::on_query_disk_info(const query_disk_info_request &req,
```

Comment:
Define it as const member functon to make sure you won't change anything.

Reply:
New commit add `register_rpc_handler(RPC_QUERY_DISK_INFO, "query_disk_info", &replica_stub::on_query_disk_info);` and 2th argument don't  match the "const member function"